webapp/graphite/dashboard/views.py

```diff
@@ -310,8 +310,8 @@ def find(request):
   results = []
 
   # Find all dashboard names that contain each of our query terms as a substring
-  for dashboard in Dashboard.objects.order_by('name'):
-    name = dashboard.name.lower()
+  for dashboard in Dashboard.objects.values('name').order_by('name'):
```

Comment:
Ah.  Yes.  Still need to have a local variable with the lower() value for comparison.  I think it's cleaner to read, though, so I changed it to that.